week-03/main.js

```diff
@@ -3,7 +3,8 @@
 // Given the following array
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
-
+list.shift();
+list.push(1);
```

Comment:
While this does add `1` which is at the beginning of the list, it's not the same `1` that was shifted off. How would you capture that value and push it to the list?